cmake/FindCPR.cmake

```diff
@@ -1,3 +1,5 @@
+find_package(CURL REQUIRED)
```

Comment:
you should not set `REQUIRED` in a find module because it always makes the build fail if curl is not found. if cpr is not `required` but `optional` you do not want to fail. better remove the `required` here and add `CURL_FOUND` to FPHSA below.

Reply:
Done.